gtsfm/scene_optimizer.py

```diff
@@ -17,17 +17,20 @@
 from dask.delayed import Delayed
 from gtsam import Pose3
 
+import gtsfm.evaluation.metrics_report as metrics_report
 import gtsfm.utils.geometry_comparisons as comp_utils
 import gtsfm.utils.metrics as metric_utils
 import gtsfm.utils.io as io_utils
 import gtsfm.utils.logger as logger_utils
 import gtsfm.utils.viz as viz_utils
+import gtsfm.utils.metrics as metrics_utils
```

Comment:
should be higher, if sorted alphabetically

Reply:
metrics_utils is imported twice I think @akshay-krishnan

Reply:
oops, fixing.